code/game/objects/items/his_grace.dm

```diff
@@ -0,0 +1,262 @@
+			if(HIS_GRACE_CONSUME_OWNER > prev_bloodthirst)
+				master.visible_message("<span class='userdanger'>[src] enters a frenzy!</span>")
+		if(HIS_GRACE_STARVING to HIS_GRACE_CONSUME_OWNER)
+			flags |= NODROP
+			if(HIS_GRACE_STARVING > prev_bloodthirst)
+				master.visible_message("<span class='boldwarning'>[src] is starving!</span>", "<span class='his_grace big'>[src]'s bloodlust overcomes you. [src] must be fed, or you will become His meal.\
+				[force_bonus < 15 ? " And still, His power grows.":""]</span>")
+				force_bonus = max(force_bonus, 15)
+		if(HIS_GRACE_FAMISHED to HIS_GRACE_STARVING)
+			flags |= NODROP
+			if(HIS_GRACE_FAMISHED > prev_bloodthirst)
+				master.visible_message("<span class='warning'>[src] is very hungry!</span>", "<span class='his_grace big'>Spines sink into your hand. [src] must feed immediately.\
+				[force_bonus < 10 ? " His power grows.":""]</span>")
+				force_bonus = max(force_bonus, 10)
+			if(prev_bloodthirst >= HIS_GRACE_STARVING)
+				master.visible_message("<span class='warning'>[src] is now only very hungry!</span>", "<span class='his_grace big'>Your bloodlust recedes.</span>")
+		if(HIS_GRACE_HUNGRY to HIS_GRACE_FAMISHED)
+			if(HIS_GRACE_HUNGRY > prev_bloodthirst)
+				master.visible_message("<span class='warning'>[src] is getting hungry.</span>", "<span class='his_grace big'>You feel [src]'s hunger within you.\
+				[force_bonus < 5 ? " His power grows.":""]</span>")
+				force_bonus = max(force_bonus, 5)
+			if(prev_bloodthirst >= HIS_GRACE_FAMISHED)
+				master.visible_message("<span class='warning'>[src] is now only somewhat hungry.</span>", "<span class='his_grace'>[src]'s hunger recedes a little...</span>")
+		if(HIS_GRACE_PECKISH to HIS_GRACE_HUNGRY)
+			if(HIS_GRACE_PECKISH > prev_bloodthirst)
+				master.visible_message("<span class='warning'>[src] is feeling snackish.</span>", "<span class='his_grace'>[src] begins to hunger.</span>")
+			if(prev_bloodthirst >= HIS_GRACE_HUNGRY)
+				master.visible_message("<span class='warning'>[src] is now only a little peckish.</span>", "<span class='his_grace big'>[src]'s hunger recedes somewhat...</span>")
+		if(HIS_GRACE_SATIATED to HIS_GRACE_PECKISH)
+			if(prev_bloodthirst >= HIS_GRACE_PECKISH)
+				master.visible_message("<span class='warning'>[src] is satiated.</span>", "<span class='his_grace big'>[src]'s hunger recedes...</span>")
+	force = initial(force) + force_bonus
+
+/obj/item/his_grace/proc/ascend()
+	if(ascended)
+		return
+	var/mob/living/carbon/human/master = loc
+	force_bonus += ascend_bonus
+	desc = "A legendary toolbox and a distant artifact from The Age of Three Powers. On its three latches engraved are the words \"The Sun\", \"The Moon\", and \"The Stars\". The entire toolbox has the words \"The World\" engraved into its sides."
+	icon_state = "his_grace_ascended"
+	item_state = "toolbox_gold"
+	ascended = TRUE
+	playsound(src, 'sound/effects/his_grace_ascend.ogg', 100)
+	if(istype(master))
```

Comment:
Should maybe have the variable right above the `istype()`? Doesn't actually change anything but it makes a bit more sense.
```suggestion
	force_bonus += ascend_bonus
	desc = "A legendary toolbox and a distant artifact from The Age of Three Powers. On its three latches engraved are the words \"The Sun\", \"The Moon\", and \"The Stars\". The entire toolbox has the words \"The World\" engraved into its sides."
	icon_state = "his_grace_ascended"
	item_state = "toolbox_gold"
	ascended = TRUE
	playsound(src, 'sound/effects/his_grace_ascend.ogg', 100)
	var/mob/living/carbon/human/master = loc
	if(istype(master))
```